packager/live_packager.cc

```diff
@@ -296,6 +304,61 @@ Status LivePackager::Package(const Segment& in, FullSegmentBuffer& out) {
   return packager.Run();
 }
 
+void AddProtectionSystem(
+    const media::ProtectionSystemSpecificInfo& pssh_info,
+    PSSHData* data) {
+  if (const std::vector<uint8_t> common_system_id(
```

Comment:
Could you please define 3 constants:  common_system_id, widevine_system_id, playready_system_id
and then use a simple if to check.

Reply:
Done, refactored